### ISSUE TITLE
5 GPU system cannot set CPU slider lower than 6.

### ISSUE THREAD

**Issue by jcoffland** [FoldingAtHome/fah-client-bastet#128]
[hucker75](https://github.com/hucker75) wrote:

Having a similar problem with a decent computer I moved the cards to, it looks like Folding hates multiple GPUs.  As per the screenshots, I have 5 GPUs, and I set it to only use them and not the CPU, as best I could.  But even though there are 5 GPUs, the minimum the slider would go for CPU was 6, hence it keeps trying to get a 1 core CPU task and fails to do so (presumably because there aren't any.

Even worse, running overnight, I get up this morning to find only 2 GPUs running, with no reason as to why.  They were all still showing up in device manager, and all still showing up in Folding's resource usage setting.  Yet it just wasn't downloading tasks for them.  Because the CPU problem was in the way?  I rebooted the machine which didn't fix it.  I closed folding and reopened it then it started behaving again.

This is 8.1.14.  May I suggest an autoupdate function?  I wasn't aware of this version coming out.

![resource usage](https://user-images.githubusercontent.com/66015018/222714613-56b6cd67-5c68-4dec-b031-ca7a0a14ed11.jpg)

![work units](https://user-images.githubusercontent.com/66015018/222714637-194c67d0-807c-4e1e-93da-9479aab22ba5.jpg)

**Comment by jcoffland** [comment]
@hucker75 wrote:

So I changed the settings to use all CPU cores, and when it tries to request work I get:

13:05:08:E ::WU25:Exception: Could not get IP address for vav22.fah.temple.edu: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server.

I tried to ping that server from another computer, and it can't find it.  Is a server down or is my client looking up an outdated or incorrect address?

EDIT: It got a WU for CPU eventually.  I guess the server is flaky, a lot needs fixed around here!

Oh well, I guess I have to give you the CPU aswell as the GPUs to keep it happy.
